lib/mM.ml

```diff
@@ -1,4 +1,7 @@
-module Heap_pages = struct
-  external total: unit -> int = "mirage_xen_heap_get_pages_total" [@@noalloc]
-  external used: unit -> int = "mirage_xen_heap_get_pages_used" [@@noalloc]
+module Heap = struct
+  external total_bytes: unit -> int64 =
+      "mirage_xen_get_heap_total_bytes" [@@noalloc]
```

Comment:
```suggestion
      "mirage_xen_get_heap_total_bytes"
```

The `caml_copy_int64` is implemented as:

```C
CAMLexport value caml_copy_int64(int64_t i)
{
  value res = caml_alloc_custom(&caml_int64_ops, 8, 0, 1);
  Int64_val(res) = i;
  return res;
}
```

The [C FFI documentation](https://caml.inria.fr/pub/docs/manual-ocaml/intfc.html#ss:c-direct-call) specifies `noalloc` to be used when:
- the code does not allocate
- the code does not raise an exception
- the code does not release the master lock

What we could otherwise do: use an int (instead of int64) -- problematic for porting to 32 bit platforms.